sonaranalyzer-dotnet/src/SonarAnalyzer.Vsix/tools-cs/install.ps1

```diff
@@ -1,28 +1,46 @@
 ﻿param($installPath, $toolsPath, $package, $project)
 
-if ('14.0', '15.0' -notcontains $project.DTE.Version)
-{
-    throw 'This package can only be installed on Visual Studio 2015 or Visual Studio 2017.'
+$invalidVsVersion = $false
+
+if ('14.0', '15.0' -notcontains $project.DTE.Version) {
```

Comment:
Do we need to put an upper bound on the VS version for the NuGet package?

Reply:
The idea was for us to control when we think it is ok to support a new version of VS.